src/services/requests/home/index.ts

```diff
@@ -336,7 +336,7 @@ const homeKfiChartCall = async (
 ): Promise<IGeckoChartResponse | undefined> => {
   try {
     const res = await api.get({
-      route: `coins/klever-finance/market_chart?vs_currency=usd&days=${
+      route: `coins/klever.org/market_chart?vs_currency=usd&days=${
```

Comment:
i think don't need to change in this lines

Reply:
you are correct, only the old one works